File: OOPs/Class and Objects/class_object.java
class Book {
    private String name, author;

    public Book(String name, String author) {
        this.name = name;
        this.author = author;
    }

    public void book_info() {
        System.out.println("Book name is " + name + " and the author is " + author +".");
    }

    public static void main(String[] args) {
        Book bigBook = new Book("Introduction to Electronics", "Iftekhar Md Shishir");
        bigBook.book_info();
    }
}

File: OOPs/Class and Objects/class_object.cpp
#include <bits/stdc++.h>
using namespace std;

using ll = long long;

class Book {
private:
    string name, author;
public:
    Book(string a, string b) {
        name = a;
        author = b;
    }
    void book_info() {
        cout << "The " << name << " " << author << " car's engine has started!" << endl;
    }
};

int main() 
{
    Book bigOne("Introduction to Electronics", "Iftekhar");
    bigOne.book_info(); 
    return 0;
}